src/Crank/CoreWCF.Perf.yml

```diff
@@ -0,0 +1,189 @@
+jobs:
+  CoreWCFPerfClient:
+    source:
+      repository: https://github.com/CoreWCF/CoreWCF.git
+      branchOrCommit: master
```

Comment:
I'm missing some crank documentation (it's a microsoft internal thingy i assume) - but i'm fairly certain this should be `main` by now instead of `master`

Reply:
> I'm missing some crank documentation (it's a microsoft internal thingy i assume) - but i'm fairly certain this should be `main` by now instead of `master`

The changes updated in this PR: https://github.com/CoreWCF/CoreWCF/pull/796, please help review.